content/lua/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,102 @@
+  - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Conditionals'
+  - 'Control FLow'
+  - 'Operators'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+---
+**Conditionals** take an expression and evaluate if it’s `true` or `false`. Code blocks run depending on the outcome of the expression. 
+
+The condition can contain logic [operators](https://www.codecademy.com/resources/docs/lua/operators) and [variables](https://www.codecademy.com/resources/docs/lua/variables). The condition can be evaluated as `true` or `false`.
+
+
+![A diagram explaining a control flow based on the validity of different statements.](https://content.codecademy.com/courses/learn-python-control-flow/control-flow.svg)
+## If Statements
+```lua
+if <condition> then
+	<code>
+end
+```
+
+`if..then` statements run a block of code only when the condition is met. If the condition is `false`, the code block inside the `if` statement does not run. All conditional statements need to have an `end` keyword to close off the statement.
+
+The following example demonstrates an `if` statement with a `true` condition
+```lua
+a = 13
+b = 16
+
+if a < b then
+	a = a + 5
+end
+
+print(a)
+
+-- This prints 18 because a WAS less than b, so the code followed through and changed the value of a. 
+```
+
+The following code block demonstrates when the condition is now `false`:
+```lua
+a = 8
+b = 10
+
+if a==b then
+	print(b + a)
+end
+-- Nothing is printed because a is not equal to b so the code statement was not executed.
+```
+
+## Else Statements
+```lua
+if <condition> then
+	<code>
+else
+	<code>
+end
+```
+
+In the event that the first condition is `false`, when there’s a subsequent `else` statement, the `else` statement will execute instead. Think of the `else` as an “if all else fails” plan. The `else` statement **must** go at the end and does not have a condition.
+
+```lua
+isNightTime = false
+
+if < isNightTime > then
+	print("Good night!")
+else
+	print("Good morning!")
+end
+-- This statement prints "Good morning!" because the if condition evaluated to false.
+```
+## ElseIf Statements
+```lua
+if <condition> then
+	<code>
+elseif <condition> then
+	<code>
+else
+	<code>
+end
+```
+
+`elseif` statements are similar to `else` statements but they can be used to create multiple **pathways** instead of an either/or format. You can have multiple `elseif` statements, each with its own condition, but the final statement has to be an `else` statement.
```

Comment:
```suggestion
`elseif` statements are similar to `else` statements but they can be used to create multiple pathways instead of an either/or format. You can have multiple `elseif` statements, each with its own condition, but the final statement has to be an `else` statement.
```

Is pathways a lua key term? I don't see anything to that effect. If not, we should remove the bolding on the term.

Reply:
It's not. I had bolded it because during my 1:1 with Jiwon we thought that there might've been a better word instead of pathways but I couldn't find one. I had bolded it as something to look back at when there was a better word.

Reply:
```suggestion
`elseif` statements are similar to `else` statements but they can be used to create multiple actions instead of an either/or format. You can have multiple `elseif` statements, each with its own condition, but the final statement has to be an `else` statement.
```